test/API/tmisc.c

```diff
@@ -4175,14 +4175,14 @@ test_misc23(void)
     hsize_t dims[]  = {10};
     hid_t   file_id = 0, group_id = 0, type_id = 0, space_id = 0, tmp_id = 0, create_id = H5P_DEFAULT,
           access_id = H5P_DEFAULT;
-#ifndef NO_OBJECT_GET_NAME
+    //#ifndef NO_OBJECT_GET_NAME
```

Comment:
@derobins We also might want to consider what capability flag some misc. functions, such as H5Iget_name here, fall under or whether that's a new flag.